File: spring-data-microservice-transaction-sample/order-service/src/main/java/sample/order/domain/model/Item.java
package sample.order.domain.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("items")
public class Item {

  @Id
  public final int itemId;
  public final String name;
  public final int price;

  public Item(int itemId, String name, int price) {
    this.itemId = itemId;
    this.name = name;
    this.price = price;
  }
}

File: scalardb-dotnet-samples/scalardb-cluster-linq-sample/Models/Item.cs
using System.ComponentModel.DataAnnotations.Schema;
using ScalarDB.Client.DataAnnotations;

namespace ScalarDbClusterLinqSample.Models;

[Table("sample.items")]
public class Item
{
    [PartitionKey]
    [Column("item_id", Order = 0)]
    public int Id { get; set; }

    [Column("name", Order = 1)]
    public string Name { get; set; } = "";

    [Column("price", Order = 2)]
    public int Price { get; set; }
}
